frontend/src/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Navbar.css';
+
+const Navbar: React.FC = () => {
+  return (
+    <nav className="navbar">
+      <ul className="nav-left">
+        <li className="nav-item">
+          <Link to="/" className="nav-link">
+            Home
+          </Link>
+        </li>
+        <li className="nav-item">
+          <Link to="/about" className="nav-link">
+            About
+          </Link>
+        </li>
+        <li className="nav-item">
+          <Link to="/games" className="nav-link">
+            Games
+          </Link>
+        </li>
+      </ul>
+      <ul className="nav-right">
+        <li className="nav-item">
+          <Link to="/signup" className="nav-link">
+            Sign Up
+          </Link>
+        </li>
+        <li className="nav-item">
+          <Link to="/login" className="nav-link">
+            Log In
+          </Link>
+        </li>
+      </ul>
+    </nav>
```

Comment:
Check out the design in Figma, on the left we need only LifeCouldBeDream text, it should work as Home button in this case